config/plugins.js

```diff
@@ -0,0 +1,55 @@
+import commonjs from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+import buble from 'rollup-plugin-buble';
+import babel from 'rollup-plugin-babel';
+import replace from 'rollup-plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+import serve from 'rollup-plugin-serve';
+import livereload from 'rollup-plugin-livereload';
+
+import babelPreset from './babel';
+
+const devServerPlugins = commandOptions => [
+  serve({
+    open: true,
+    contentBase: '.',
+    openPage: '/examples/one-page-umd.html',
```

Comment:
As I mentioned in channel, dev server target is `ROOT/index.js` supporting either the mdx or js examples file imports. The current webpack html template is at `ROOT/index.html`.